core/src/test/java/com/linecorp/armeria/common/HttpDataTest.java

```diff
@@ -66,6 +66,14 @@ void wrapRangeNoOffset() {
             assertThat(data.array()).containsExactly(1, 5);
         }
 
+        @Test
+        void wrapRangeFull() {
+            final HttpData data = HttpData.wrap(payload, 0, 4);
```

Comment:
Could update the test case to make sure non-zero offset works.

Reply:
`wrapRange` above has non-zero offset is there another case I missed?